dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletionWithData/AzureChatCompletionWithData.cs

```diff
@@ -256,7 +256,9 @@ private T DeserializeResponse<T>(string body)
             Messages = this.GetMessages(chat)
         };
 
-        return HttpRequest.CreatePostRequest(this.GetRequestUri(), payload);
+        var httpRequest = HttpRequest.CreatePostRequest(this.GetRequestUri(), payload);
+        httpRequest.Headers.TryAddWithoutValidation("x-policy-id", this._config.ResponsibleAIPolicy);
```

Comment:
Does it make sense to add the same HTTP header to [AzureChatCompletion.cs](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletion/AzureChatCompletion.cs)?